.github/workflows/migration-engine.yml

```diff
@@ -61,21 +63,46 @@ jobs:
             ~/.cargo/registry
             ~/.cargo/git
             target
-          key: migration-engine-${{ runner.os }}-cargo-${{ matrix.database.name }}-${{ hashFiles('**/Cargo.lock') }}
+          key: me-ie-${{ runner.os }}-cargo-${{ matrix.database.name }}-${{ hashFiles('**/Cargo.lock') }}
 
-      - run: timeout 40m cargo test
-        if: ${{ matrix.database.name != 'vitess_5_7' && matrix.database.name != 'vitess_8_0' }}
-        working-directory: migration-engine/migration-engine-tests
+      - run: cargo test -p introspection-engine-tests
+        if: ${{ !matrix.database.is_vitess }}
+        env:
+          CLICOLOR_FORCE: 1
+          TEST_DATABASE_URL: ${{ matrix.database.url }}
+
+      - run: cargo test -p sql-schema-describer
+        if: ${{ !matrix.database.is_vitess }}
         env:
           CLICOLOR_FORCE: 1
           TEST_DATABASE_URL: ${{ matrix.database.url }}
 
-      - run: cargo test -- --test-threads=1
-        if: ${{ matrix.database.name == 'vitess_5_7' || matrix.database.name == 'vitess_8_0' }}
+      - run: cargo test -p migration-engine-tests
+        if: ${{ !matrix.database.is_vitess }}
         working-directory: migration-engine/migration-engine-tests
         env:
           CLICOLOR_FORCE: 1
           TEST_DATABASE_URL: ${{ matrix.database.url }}
+
+      - run: cargo test -p introspection-engine-tests -- --test-threads=1
+        if: ${{ matrix.database.is_vitess }}
+        env:
+          CLICOLOR_FORCE: 1
+          TEST_DATABASE_URL: ${{ matrix.database.url }}
+          TEST_SHADOW_DATABASE_URL: ${{ matrix.database.shadow_database_url }}
+
+      - run: cargo test -p sql-schema-describer -- --test-threads=1
```

Comment:
We should not test this here until describer can do soft resets, or vitess can create/drop databases without crashing.

Reply:
Will comment out with an explanation for now